Artifacts/windows-slack/Artifactfile.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2015-01-01/dtlArtifacts.json",
+  "title": "Slack",
+  "description": "Installs Slack using the Chocolatey package manager",
+  "tags": [
+    "Windows",
+    "Slack"
+  ],
+  "iconUri": "https://chocolatey.org/content/packageimages/slack.1.0.0.png",
+  "targetOsType": "Windows",
+   "parameters": {
+        "installUsername": {
```

Comment:
The key problem is that Slack (with this artifact) does not install synchronous . Slack will be installed on the machine only after the user (which specified in artifcat parameters), connect via RDP to this machine. It is a feature of the msi installer that is used in the choco package. Therefore, in this artifact I cannot use temporary user.